### ISSUE TITLE
Option/flag to run async tests in parallel 

### ISSUE THREAD

**Issue by checkerschaf** [denoland/deno#7615]
Hi,
a really good way to improve test performance would be to add the flag/option to run async tests in parallel. Currently this is not possible but it would increase test speed of independent (well-written) tests A LOT.
I understand that this might break some existing tests so I would either go with the `--unstable` flag or add a new option like `--parallel`.

What do you think about this?

**Comment by kitsonk** [comment]
Duplicate of #6559

Please use the search function before opening new issues.
